device/protocol_mqtt_v1.go

```diff
@@ -203,9 +212,8 @@ func (d *Device) SendIndividualMessageWithTimestamp(interfaceName, interfacePath
 	} else {
 		return fmt.Errorf("Interface %s not registered", interfaceName)
 	}
-
 	// We are good to go. Let's send the message.
-	return d.sendMqttV1MessageInternal(iface, interfacePath, value, timestamp)
+	return d.enqueueMqttV1Message(iface, interfacePath, value, timestamp)
```

Comment:
Since we're note sending the message right away anymore, update the documentation to indicate the new behaviour (including the fact that the call can be blocking if the message channel is full etc)

Reply:
Also remember to update the doc for the other function without the timestamp in the signature.